libs/api/domains/communications/src/lib/emailTemplates/serviceWebForms.ts

```diff
@@ -4,55 +4,55 @@ import { environment } from '../environments/environment'
 
 type StringOrNull = string | null
 
+// Fjölskyldumál
+const FJOLSKYLDUMAL = '4vQ4htPOAZvzcXBcjx06SH'
+
+// Skírteini
+const SKIRTEINI = '7nWhQCER920RakQ7BZpEmV'
+
+// Andlát og dánarbú
+const ANDLAT_OG_DANARBU = '2TkJynZlamqTHdjUziXDG0'
+
+// Þinglýsingar, staðfestingar og skráningar
+const THINGLYSINGAR = '6K9stHLAB2mEyGqtqjnXxf'
+
+// Gjöld og innheimta
+const GJOLD_OG_INNHEIMTA = '5u2M09Kw3p1Spva1GSuAzB'
+
+// Löggildingar
+const LOGGILDINGAR = 'WrQIftmx61sHJMoIr1QRW'
+
+// Vottorð
+const VOTTORD = '76Expbwtudon1Gz5lrKOit'
+
+// Lögráðamál
+const LOGRADAMAL = '4tvRkPgKP3kerbyRJDvaWF'
+
+// Önnur þjónusta sýslumanna
+const ONNUR_THJONUSTA_SYSLUMANNA = '4LNbNB3GvH3RcoIGpuZKhG'
+
+// Leyfi
+const LEYFI = '7HbSNTUHJReJ2GPeT1ni1C'
+
+// Fullnustugerðir
+const FULLNUSTUGERDIR = '7LkzuYSzqwM7k8fJyeRbm6'
+
+// Default/fallback email address
+const DEFAULT = 'default'
+
```

Comment:
Have you considered using enum?

Reply:
I've never used enums in TypeScript before but it'd probably be a smarter move: https://www.typescriptlang.org/docs/handbook/enums.html

Reply:
![image](https://user-images.githubusercontent.com/43557895/168267434-4462ee43-e9cd-4ab8-a702-ad8db0138c18.png)
This is way better :)

Reply:
@RunarVestmann

Reply:
I addressed this in another PR: https://github.com/island-is/island.is/pull/7410